package com.test.kotlin

import io.kotlintest.specs.StringSpec

class KotlinExceptionTest : StringSpec() {

    /**
     * try-catch-finally 구문은 java와 다를게 없다.
     * 다른점은 이 또한 Expession으로 동작하는 점이다.
     *
     *
     * Kotlin에서는 CheckedException과 UnCheckedException을 구분하지 않는다.
     * 모두 UnCheckedException으로 간주한다.
     */
}